Raven.Database/Bundles/ScriptedIndexResults/ScriptedIndexResultsIndexTrigger.cs

```diff
@@ -71,7 +71,7 @@ public override void OnIndexEntryCreated(string entryKey, Document document)
             {
                 if (created.ContainsKey(entryKey) == false)
                 {
-                    created[entryKey] = new List<RavenJObject>();
+                    created.Add(entryKey, new List<RavenJObject>());
```

Comment:
How can this fix any issue? For that matter, the error said get_Item, you are changing a set_Item